kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/convert/KafkaDataBuilder.java

```diff
@@ -52,7 +53,9 @@ public static Field buildKafkaDataField(String kafkaDataFieldName) {
                 .setMode(com.google.cloud.bigquery.Field.Mode.NULLABLE);
         Field insertTimeField = insertTimeBuilder.build();
 
-        return Field.newBuilder(kafkaDataFieldName, LegacySQLTypeName.RECORD,
+        return Field.newBuilder(
+                kafkaDataFieldName,
+                LegacySQLTypeName.RECORD,
```

Comment:
Can we revert this whitespace-only change? Might make a `pint merge` harder in the future.